src/style.less

```diff
@@ -101,11 +108,54 @@ h3 .superscript {
 .content {
   padding: 0.5rem 2rem;
 
+  select {
+    max-width: 15rem;
+    margin-bottom: 1rem;
+    padding: 0.75rem 2.5rem 0.75rem 1rem;
+    border: none;
+    border-radius: 1px;
+    background: lighten(@secondary, 3%);
+    background-image: url('./images/chevron-down.svg');
+    background-position: right 0.75rem top 50%, 0 0;
+    background-repeat: no-repeat;
+    color: @body;
+    font-family: @body-font;
+    font-size: 1rem;
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+
+    &:focus {
+      outline: none;
+    }
+
+    option {
+      &:hover {
```

Comment:
```suggestion
      &:hover, &:focus {
```

Reply:
I don't think the option:hover, &:focus styling is needed at all